recipes/pythonqwt/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "PythonQwt" %}
+{% set version = "0.5.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PierreRaybaut/PythonQwt/archive/v{{ version }}.tar.gz
+  sha256: ca3b4e34f51ade4681b99d73b5a9134498fee84a832e36cfd8f400b9acc07723
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - setuptools
+    - python
+    - pyqt
+    - spyder
+    - numpy
+    - pip
+  run:
+    - python
+    - pyqt
+    - spyder
+    - numpy
+
+test:
+  imports:
+    - qwt
+    - qwt.tests
+    - qwt.plot
+    - qwt.qt
+
+about:
+  home: http://pierreraybaut.github.io/PythonQwt/
+  license: LGPL
```

Comment:
It looks like it's licensed under some custom flavor of LGPL, but also MIT. Not sure how best to handle it here.

Reply:
Looking at it, LGPL is probably reasonable, since it's the most restrictive (other than some parts of the tests).